modules/model_loader.py

```diff
@@ -20,7 +20,12 @@ def load_file_from_url(
         file_name = os.path.basename(parts.path)
     cached_file = os.path.abspath(os.path.join(model_dir, file_name))
     if not os.path.exists(cached_file):
-        print(f'Downloading: "{url}" to {cached_file}\n')
-        from torch.hub import download_url_to_file
-        download_url_to_file(url, cached_file, progress=progress)
-    return cached_file
+        try:
+
+            print(f'Downloading: "{url}" to {cached_file}\n')
+            from torch.hub import download_url_to_file
+            download_url_to_file(url, cached_file, progress=progress)
+        except Exception as e:
+            print(f"Failed to download {url} to {cached_file}: {e}")
```

Comment:
inconsistency in quotes, "{url}" used above but here it's only {url}. Please use "{url}".

```suggestion
            print(f"Failed to download "{url}" to {cached_file}, reason: {e}")
```